golang/x/relay/client/rest/query.go

```diff
@@ -65,6 +65,34 @@ func isAncestorHandler(cliCtx context.CLIContext, storeName string) http.Handler
 	}
 }
 
+// handler function for getRelayGenesis queries. parses arguments from url string, and passes them through
+// as a QueryParamsGetRelayGenesis struct
+func getRelayGenesisHandler(cliCtx context.CLIContext, storeName string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		res, _, err := cliCtx.QueryWithData("custom/relay/getrelaygenesis", nil)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusNotFound, err.Error())
+			return
+		}
+
+		rest.PostProcessResponse(w, cliCtx, res)
+	}
+}
+
+// handler function for getLastReorgLCA queries. parses arguments from url string, and passes them through
+// as a QueryParamsGetLastReorgLCA struct
```

Comment:
comment is not accurate, as there are no params

Reply:
This is true